testsuite/features/secondary/min_ssh_tunnel.feature

```diff
@@ -4,13 +4,13 @@
 Feature: Register a salt system to be managed via SSH tunnel
 
   Scenario: Delete the salt minion for SSH reverse bootstrap
```

Comment:
We still have some mixed names "SSH reverse bootstrap" , "minion for push via SSH tunnel", "salt minion after SSH push tests" :thinking:

Reply:
Indeed.  Changed `reverse` by `tunnel` in those steps.